contentcuration/contentcuration/frontend/channelEdit/components/PublishModal/ChannelMetadata.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+
+  <div>
+    <VProgressCircular v-if="loading" indeterminate size="16" color="grey lighten-1" />
+    <template v-else>
+      <h2 class="subheading">
+        <span v-if="metadata.channelVersion">
+          {{ $tr('versionText', { version: metadata.channelVersion }) }}
+        </span>
+        <span v-else>
+          {{ $tr('unpublishedText') }}
+        </span>
+      </h2>
+      <div class="body-2 grey--text metadata">
+        <span>
+          {{ $tr('publishingSizeText', { count: metadata.resourceCount }) }}
+        </span>
+        <span>
+          {{ formatFileSize(metadata.size) }}
+        </span>
+        <span>
+          {{ metadata.languageName }}
+        </span>
+      </div>
+    </template>
+  </div>
+
+</template>
+
+<script>
+
+  import { fileSizeMixin } from 'shared/mixins';
+
+  export default {
+    name: 'ChannelMetadata',
+    mixins: [fileSizeMixin],
+    props: {
+      metadata: {
+        type: Object,
+        required: true,
+      },
+      loading: {
+        type: Boolean,
+      },
+    },
+    $trs: {
+      versionText: 'Version {version}',
+      unpublishedText: 'Unpublished',
+      publishingSizeText: '{count, plural, =1 {# resource} other {# resources}}',
```

Comment:
If we keep an otherwise unused `PublishModal` component around, we can probably borrow this and the other strings here using `crossComponentTranslator` until we manage to get a new set of translations done.